tests/s2_inference/test_encoding.py

```diff
@@ -49,12 +51,19 @@ def test_vectorize(self):
 
             for sentence in sentences:
                 output_v = vectorise(name, sentence, model_properties, device, normalize_embeddings=True)
-
                 assert _check_output_type(output_v)
-
                 output_m = model.encode(sentence, normalize=True)
-
                 assert abs(torch.FloatTensor(output_m) - torch.FloatTensor(output_v)).sum() < eps
+                for vector in output_v:
+                    assert abs(torch.linalg.norm(np.array(vector)) - 1) < 1e-5
+
+                output_v_unnormalised = vectorise(name, sentence, model_properties, device, normalize_embeddings=False)
+                assert _check_output_type(output_v)
+                output_m_unnormalised = model.encode(sentence, normalize=False)
+                assert abs(torch.FloatTensor(output_v_unnormalised) - torch.FloatTensor(output_m_unnormalised)).sum() < eps
+
+                for vector in output_v_unnormalised:
+                    assert abs(torch.linalg.norm(np.array(vector)) - 1) > 1e-5
```

Comment:
Hello @wanliAlex , there are a few issues with this assertion,
1. the norm method takes a Tensor param
2. Some models normalise the embeddings by default.